.revive.toml

```diff
@@ -0,0 +1,66 @@
+# Sets the default severity to "error"
+severity = "error"
+
+# Sets the default failure confidence. This means that linting errors
+# with less than 0 confidence will be ignored.
+confidence = 0
+
+# https://github.com/mgechev/revive#configuration
+
+# Recommended configuration, minus [rule.exported] and [rule.package-comments]
+[rule.blank-imports]
+[rule.context-as-argument]
+[rule.context-keys-type]
+[rule.dot-imports]
+[rule.empty-block]
+[rule.error-return]
+[rule.error-strings]
+[rule.error-naming]
+[rule.errorf]
+#[rule.exported]
+[rule.if-return]
+[rule.increment-decrement]
+[rule.indent-error-flow]
+#[rule.package-comments]
```

Comment:
There are hundreds of "exported var needs comment" or "package needs comment" annotations. I'm disabling them to reduce noise.